tests/Nelmio/Alice/Fixtures/LoaderTest.php

```diff
@@ -1707,30 +1707,40 @@ public function testArrayParametersLoading()
 
     public function testCompositeParametersLoading()
     {
-        $this->markTestSkipped('Parameters cannot be composite yet.');
-        $res = $this->createLoader()->load(__DIR__ . '/Files/parameters/composite.yml');
+        $objects = $this->createLoader()->load(__DIR__ . '/Files/parameters/composite.yml');
 
-        $this->assertCount(1, $res);
+        $this->assertCount(2, $objects);
 
-        $user = $res['user0'];
+        $user = $objects['user0'];
+        $this->assertInstanceOf(self::USER, $user);
+        //$this->assertEquals('NaN Bat!', $user->username); Not supported yet
```

Comment:
I would like to keep it to show what *it should be*... but I can understand that people don't want comments